src/components/MultiHopTrade/MultiHopTrade.tsx

```diff
@@ -31,11 +37,17 @@ const TradeRouteEntries = [
 export type TradeCardProps = {
   defaultBuyAssetId?: AssetId
   isCompact?: boolean
+  isRewritingUrl?: boolean
 }
 
 type MatchParams = {
   chainId?: string
   assetSubId?: string
+  fromAssetSubId?: string
+  fromChainId?: string
```

Comment:
Looks like these two are unused according to https://github.com/shapeshift/web/blob/d789621d0fda1444f54da7cd716cd0c06d1f3d2c/src/components/MultiHopTrade/MultiHopTrade.tsx#L64

Reply:
Nice catch, changed in [04673b4](https://github.com/shapeshift/web/pull/8708/commits/04673b4e433c4d17da3ee076b00c2ca0d352a2d1)